### PR TITLE
CDAP-6321 Master Service Bind Port Not Configurable

### PR DESCRIPTION
**Pull request by bennythomps**

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -596,6 +604,14 @@
...
 
   <property>
+    <name>dataset.service.bind.port</name>
+    <value>0</value>
+    <description>
+      Dataset service bind port
+    </description>
+  </property>
+
+  <property>
```
Comment (anew) [inline cdap-common/src/main/resources/cdap-default.xml:622]: The travis build failed... looks like we also need a default for dataset.executor.bind.port.

**Comment by anew** [comment]
Other than the missing default for "dataset.executor.bind.port", changes look good to me. @bennythomps, can you fix that and hopefully then the Travis build should pass. Thanks

**Comment by anew** [comment]
The travis build failed but I cannot explain it. So I ran a build in Bamboo with these changes, and it passed: http://builds.cask.co/browse/CDAP-DUT4459-1

LGTM and merging now.

**Comment by anew** [comment]
I merge and reverted this. It is LGTM

**Comment by anwar6953** [comment]
Why revert it?

**Comment by anew** [comment]
It is an external contribution. I cannot merge until I know that a CLA was signed. @nitinmotgi I believe you are the one to merge this. I think you can simply revert the PR that reverted this.

**Comment by nitinmotgi** [comment]
Merge it!

**Comment by bennythomps** [comment]
This is breaking critical functionality for us. Is there a reason it's still not merged @anew?

**Comment by anwar6953** [comment]
Hey @bennythomps. The change was actually merged to develop, but didn't make it to release/3.5, since the release branch was branched prior to @nitinmotgi's PR (https://github.com/caskdata/cdap/pull/6192).

I opened a PR to cherry-pick this change onto the release/3.5 branch:
https://github.com/caskdata/cdap/pull/6392

**Comment by bennythomps** [comment]
Thanks @anwar6953 !

**Pull request merged**